services/account/apps/api/src/services/handles.service.ts

```diff
@@ -17,4 +17,13 @@ export class HandlesService {
     }
     throw new Error('Invalid msaId.');
   }
+
+  async getExpiration(): Promise<number> {
+    const lastFinalizedBlockNumber = await this.blockchainService.getLatestFinalizedBlockNumber();
+    return lastFinalizedBlockNumber + 10;
```

Comment:
10 blocks may not be long enough to enable signing of this transaction

100@ 6 seconds or 50 @ 12 seconds should probably be a default constant from the config service
```suggestion
    return lastFinalizedBlockNumber + 100;
```
or if we add `@Query('expirationTime') expirationTime?: number,` to the endpoint.
```suggestion
    return lastFinalizedBlockNumber + expirationTime/SECS_PER_BLOCK; //Pseudocode
```